### PR TITLE
Manager -> pilot rename

### PR DESCRIPTION
**Pull request by ldemailly**
git ls-files | grep -v tests/e2e|grep -v bookinfo| xargs -n 1 sed -i .bak -e 's/Manager/Pilot/g'
git ls-files | grep -v tests/e2e|grep -v bookinfo| xargs -n 1 sed -i .bak -e 's/manager/pilot/g'

skipping bookinfo and tests/e2e(*) which have other managers

*: checking the rest of e2e by hand now

### CODE REVIEW

File: install/kubernetes/templates/istio-manager.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Manager service for discovery
+# Pilot service for discovery
```
Comment (kyessenov) [inline install/kubernetes/templates/istio-manager.yaml]: rename the file itself

Comment (ldemailly) [inline install/kubernetes/templates/istio-manager.yaml]: done

File: install/updateVersion.sh [hunk 2]
```diff
@@ -43,17 +43,17 @@ source "$VERSION_FILE" || error_exit "Could not source versions"
 while getopts :ci:m:x:C: arg; do
   case ${arg} in
     i) ISTIOCTL_URL="${OPTARG}";;
-    m) MANAGER_HUB_TAG="${OPTARG}";; # Format: "<hub>,<tag>"
+    m) PILOT_HUB_TAG="${OPTARG}";; # Format: "<hub>,<tag>"
```
Comment (sebastienvas) [inline install/updateVersion.sh]: do you want to chage m) to p) ?

**Comment by ldemailly** [comment]
so the build fails with some linter error ? not sure I'm parsing the error correctly ?

**Comment by sebastienvas** [comment]
Just run bin/fmt.sh

File: install/updateVersion.sh [hunk 3]
```diff
@@ -43,17 +43,17 @@ source "$VERSION_FILE" || error_exit "Could not source versions"
 while getopts :ci:m:x:C: arg; do
```
Comment (sebastienvas) [inline install/updateVersion.sh]: CHange the :ci:m:x:C: t :ci:p:x:C:

File: install/updateVersion.sh [hunk 4]
```diff
@@ -43,7 +43,7 @@ source "$VERSION_FILE" || error_exit "Could not source versions"
...
```
Comment (frankbu) [inline install/updateVersion.sh]: This "m" needs to be changed to "p" also. 
```
while getopts :ci:p:x:C: arg; do
```

File: tests/README.md [hunk 5]
```diff
@@ -5,12 +5,12 @@ This directory contains scripts for running a full Istio end-to-end integration
...
 
 The [kubeTest.sh](kubeTest.sh) script allows a user to run the integration test using
-any combination of `istioctl`, `manager`, and `mixer` versions. Developers should use
+any combination of `istioctl`, `pilot`, and `mixer` versions. Developers should use
 it to test their changes before creating PRs.
 
 ### Options
 
-* `-m <hub>,<tag>` specify a manager image to use
+* `-m <hub>,<tag>` specify a pilot image to use
```
Comment (sebastienvas) [inline tests/README.md]: Change m  to p

File: install/updateVersion.sh (hunk 3 above)
Comment (ldemailly) [inline install/updateVersion.sh]: oh duh me ! thx!

**Pull request merged**